learn/multi_search/implement_sharding.mdx

```diff
@@ -93,32 +98,54 @@ curl \
 
 Meilisearch processes searches on the remote that corresponds to `self` locally instead of making a remote request.
 
+### Enabling sharding
+
+Finally enable the automatic sharding of documents by Meilisearch on all instances:
+
+```sh
+curl \
+  -X PATCH 'MEILISEARCH_URL/network' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "sharding": true
+  }'
+```
+
 ### Adding or removing an instance
 
 Changing the topology of the network involves moving some documents from an instance to another, depending on your hashing scheme.
 
 As Meilisearch does not provide atomicity across multiple instances, you will need to either:
 
-1. accept search downtime while migrating documents  
-2. accept some documents will not appear in search results during the migration  
+1. accept search downtime while migrating documents
+2. accept some documents will not appear in search results during the migration
 3. accept some duplicate documents may appear in search results during the migration
 
 #### Reducing downtime
 
 If your disk space allows, you can reduce the downtime by applying the following algorithm:
 
-1. Create a new temporary index in each remote instance  
-2. Compute the new instance for each document  
-3. Send the documents to the temporary index of their new instance  
-4. Once Meilisearch has copied all documents to their instance of destination, swap the new index with the previously used index  
-5. Delete the temporary index after the swap  
-6. Update network configuration and search queries across all instances  
+1. Create a new temporary index in each remote instance
+2. Compute the new instance for each document
+3. Send the documents to the temporary index of their new instance
+4. Once Meilisearch has copied all documents to their instance of destination, swap the new index with the previously used index
+5. Delete the temporary index after the swap
+6. Update network configuration and search queries across all instances
+
+## Create indexes
+
+Create the same empty indexes with the same settings on all instances.
+Keeping the settings and indexes in sync is important to avoid errors and unexpected behavior, though not strictly required.
+
+## Add documents
+
+Send documents to any instance to have them replicated to the other instances.
```

Comment:
I remembered we said people should send to one shard, but my memory is not great today 😂

Reply:
the answer is a bit subtle here.

1. In v1.19, as long as the client waits for the engine's response before sending another task, they can send their documents to any instance.
2. **However**, we might not keep this property in future versions (to be able to dynamically update the network topology), so it might be best to already advise using a single node for writes.